docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -291,6 +291,33 @@ The content provided in your app must be easily understandable to all users. Tha
 
 ---
 
+## Distinguishable (WCAG 1.4)
+
+### Reflow (WCAG 1.4.10 - Level AA)
+
+The reflow is basically another name for responsible design in this context. The content should be able to reflow from one screen size to another without losing information or functionality.
+
+#### ✅ Success criteria
+
+Every component on the screen should be able to reflow from one screen size to another, without any content disappearing or becoming obscured. The same applies when the device orientation is changed or the content is enlarged in any way (increased font or display size in the device settings).
+
+The above can be achieved by using:
+- Components with variable size (i.e. by not restricting the maximum height/width).
+- Scrollable containers when content is laid out linearly. In XML, use `ScrollView` (or `NestedScrollView`) for any content that can potentially go off screen on smaller screen sizes. In Compose, the same can be achieved by using a `LazyColumn/Row` or just a regular `Column/Row` with a `vertical/horizontalScroll()` modifier if lazy behavior is not needed.
+- Layouts that allow the content to flow and wrap into multiple lines if it becomes too wide to fit the screen. In XML, this can be achieved by using the [`Flow`](https://developer.android.com/reference/androidx/constraintlayout/helper/widget/Flow) layout. In Compose, there are [`FlowRow/Column`](https://developer.android.com/develop/ui/compose/layouts/flow) for the same purpose.
```

Comment:
Yep, they are. But I don't think there is any good alternative that is non-experimental, at least not an official one. 😞 Should we have some kind of a disclaimer here? I hope that they will become stable soon, since they were added in, I think, 1.4.0, but who knows. 😕

Reply:
Added a disclaimer. ✅ I'll try to keep an eye on the releases and remove it when they become stable.